File: programmers/js/new/level1/2016년.js
function solution(a, b) {
    a--;
    let days = 0;
    const daysInMonth = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
    for (let i = 0; i < a; ++i) {
        days += daysInMonth[i];
    }
    days += b;
    const weekDays = ["THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"];
    return weekDays[days % 7];
}

File: programmers/python/level1/2016년.py
def solution(a, b):
    days_by_month = [0,31,29,31,30,31,30,31,31,30,31,30,31]
    total_days = b
    for i in range(1, a):
        total_days += days_by_month[i]
    week_days = ['THU', 'FRI', 'SAT', 'SUN', 'MON', "TUE", "WED"]
    return week_days[total_days % 7]